tendermint-lite/src/main.rs

```diff
@@ -0,0 +1,134 @@
+use tendermint::hash::Algorithm;
```

Comment:
Perhaps it would be less ambiguous to import `tendermint::hash` and reference `Algorithm` as `hash::Algorithm`?

Reply:
Addressed in 712e673

@tarcieri Is it necessary to expose this param at all at this point if the enum has only one variant?

Reply:
I'd say it's both more explicit and good futureproofing in the event you do ever want to support additional hash algorithms. I'll cite `git` as a project which didn't do this which also still hasn't managed to migrate to a new hash algorithm.

That said I doubt SHA-256 will ever be catastrophically broken.